models/UserSearch.php

```diff
@@ -95,6 +99,7 @@ public function search($params)
 
         $query->andFilterWhere(['like', 'username', $this->username])
             ->andFilterWhere(['like', 'email', $this->email])
+            ->andFilterWhere(['user.id' => $this->id])
```

Comment:
You should not use it like that.
The model may have been overwritten and be using another table name.